L1Trigger/L1THGCal/python/hgcalConcentratorProducer_cfi.py

```diff
@@ -98,21 +98,36 @@
 
 
 mixedbcstc_conc_proc = cms.PSet(ProcessorName  = cms.string('HGCalConcentratorProcessorSelection'),
-                          Method = cms.string('mixedBestChoiceSuperTriggerCell'),
+                          Method = cms.vstring('bestChoiceSelect','superTriggerCellSelect','superTriggerCellSelect'),
                           NData = cms.vuint32(bestchoice_ndata_centralized),
                           coarsenTriggerCells = cms.bool(False),
                           fixedDataSizePerHGCROC = cms.bool(False),
                           type_energy_division = cms.string('superTriggerCell'),# superTriggerCell,oneBitFraction,equalShare
                           stcSize = cms.vuint32(STC_SIZE),
                           ctcSize = cms.vuint32(CTC_SIZE),
-                          supertccompression = superTCCompression_proc.clone(),
+                          superTCCompression = superTCCompression_proc.clone(),
+                          coarseTCCompression = coarseTCCompression_proc.clone(),
+                          superTCCalibration = vfe_proc.clone(),
+                          )
+
+custom_conc_proc = cms.PSet(ProcessorName  = cms.string('HGCalConcentratorProcessorSelection'),
+                          Method = cms.vstring('bestChoiceSelect','superTriggerCellSelect','superTriggerCellSelect'),
+                          NData = cms.vuint32(bestchoice_ndata_centralized),
+                          threshold_silicon = cms.double(2.), # MipT
+                          threshold_scintillator = cms.double(2.), # MipT
+                          coarsenTriggerCells = cms.bool(False),
+                          fixedDataSizePerHGCROC = cms.bool(False),
+                          type_energy_division = cms.string('superTriggerCell'),# superTriggerCell,oneBitFraction,equalShare
+                          stcSize = cms.vuint32(STC_SIZE),
+                          ctcSize = cms.vuint32(CTC_SIZE),
+                          superTCCompression = superTCCompression_proc.clone(),
                           coarseTCCompression = coarseTCCompression_proc.clone(),
                           superTCCalibration = vfe_proc.clone(),
                           )
```

Comment:
Is there a difference between `mixedbcstc_conc_proc` and `custom_conc_proc`?

Reply:
not really, so I have removed `mixedbcstc_conc_proc` here